grafana/overlays/emea/devconfus2021/opf-monitoring/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-monitoring
+generatorOptions:
+  disableNameSuffixHash: true
+configMapGenerator:
+  - name: grafana-config-overrides
+    literals:
+      - GF_AUTH_GENERIC_OAUTH_ROLE_ATTRIBUTE_STRICT=true
```

Comment:
I don't think we need this. @4n4nd thoughts?

Reply:
yeah I don't think we need it. let's keep permissions relaxed since this is a temp cluster

Reply:
Resolved in latest commit.